src/utils.ts

```diff
@@ -56,7 +56,7 @@ export class Utils {
   /** convert a potential selector into actual list of html elements. optional root which defaults to document (for shadow dom) */
   static getElements(els: GridStackElement, root: HTMLElement | Document = document): HTMLElement[] {
     if (typeof els === 'string') {

```

Comment:
I would just return if root is null instead of all these checks...

Reply:
Do you mean return undefined since root is undefined on server side? 

I'll change it to that.

Reply:
yes if not needed, or better fix as I suggested.

Reply:
done, i have reverted the file.